<launch>
    <arg name="camera_name"     default="d435i"/>
    <arg name="output_folder" default="/media/$(env USER)/Samsung"/>

    <node pkg="rosbag" type="record" name="record_bag" 
        args="record /$(arg camera_name)/color/image_raw/compressed
                /$(arg camera_name)/color/camera_info
                /$(arg camera_name)/aligned_depth_to_color/image_raw
                /$(arg camera_name)/aligned_depth_to_color/camera_info
                /$(arg camera_name)/depth/color/points
                /tf
                /tf_static
                --lz4
                -o $(arg output_folder)/camera"
    />

</launch>